### PR TITLE
Refactor graphql

### PR DESCRIPTION
**Pull request by mgs95**
- Replace .graphql files with .ts files
 - Remove client queries

### CODE REVIEW

File: app/ui/src/Components/ProjectsBar/components/ProjectsFilter/ProjectsFilter.tsx [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
 import capitalize from 'lodash.capitalize';
-import { loader } from 'graphql.macro';
 import styles from './ProjectsFilter.module.scss';
 import useProjectFilters from 'Graphql/client/hooks/useProjectFilters';
-import { useQuery } from '@apollo/client';
+import { useQuery, useReactiveVar } from '@apollo/client';
 
-const GetProjectsQuery = loader('Graphql/queries/getProjects.graphql');
+import GetProjectsQuery from 'Graphql/queries/getProjects';
+import { projectFilters } from '../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Components/ProjectsBar/components/ProjectsFilter/ProjectsFilter.tsx:12]:
```suggestion
import { projectFilters } from 'Graphql/client/cache';
```

File: app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.tsx [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
 import React from 'react';
-import { loader } from 'graphql.macro';
 import styles from './ProjectsOrder.module.scss';
 import useProjectFilters from 'Graphql/client/hooks/useProjectFilters';
-import { useQuery } from '@apollo/client';
+import { useQuery, useReactiveVar } from '@apollo/client';
 
-const GetProjectsQuery = loader('Graphql/queries/getProjects.graphql');
+import GetProjectsQuery from 'Graphql/queries/getProjects';
+import { projectFilters } from '../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.tsx:11]:
```suggestion
import { projectFilters } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/NewProject/NewProject.tsx [hunk 3]
```diff
@@ -17,15 +13,15 @@ import Repository from './pages/Repository/Repository';
...
 import Stepper from 'Components/Stepper/Stepper';
 import Summary from './pages/Summary/Summary';
 import cx from 'classnames';
 import styles from './NewProject.module.scss';
 import useNewProject from 'Graphql/client/hooks/useNewProject';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
 import { Prompt } from 'react-router-dom';
 import useUnloadPrompt from 'Hooks/useUnloadPrompt/useUnloadPrompt';
+import { newProject } from '../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/NewProject/NewProject.tsx:24]:
```suggestion
import { newProject } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/NewProject/pages/RepositoryDetails/RepositoryDetails.tsx [hunk 4]
```diff
@@ -1,29 +1,17 @@
...
-  GetNewProject,
-} from 'Graphql/client/queries/getNewProject.graphql';
-
 import React from 'react';
 import { RepositoryType } from 'Graphql/types/globalTypes';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
 import ExternalRepository from './components/ExternalRepository/ExternalRepository';
 import InternalRepository from './components/InternalRepository/InternalRepository';
+import { newProject } from '../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/NewProject/pages/RepositoryDetails/RepositoryDetails.tsx:6]:
```suggestion
import { newProject } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/NewProject/pages/RepositoryDetails/components/InternalRepository/InternalRepository.tsx [hunk 5]
```diff
@@ -1,16 +1,13 @@
...
+import React from 'react';
 
 import { CONFIG } from 'index';
 import IconLink from '@material-ui/icons/Link';
 import styles from './InternalRepository.module.scss';
 import useNewProject from 'Graphql/client/hooks/useNewProject';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
 import { getErrorMsg, replaceAll } from 'Utils/string';
+import { newProject } from '../../../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/NewProject/pages/RepositoryDetails/components/InternalRepository/InternalRepository.tsx:10]:
```suggestion
import { newProject } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/NewProject/pages/Summary/Summary.tsx [hunk 6]
```diff
@@ -1,16 +1,12 @@
...
 
 import { CONFIG } from 'index';
 import CopyToClipboard from 'Components/CopyToClipboard/CopyToClipboard';
 import IconLink from '@material-ui/icons/Link';
 import { RepositoryType } from 'Graphql/types/globalTypes';
-import { SpinnerCircular } from 'kwc';
 import styles from './Summary.module.scss';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
+import { newProject } from '../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/NewProject/pages/Summary/Summary.tsx:9]:
```suggestion
import { newProject } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/ProjectCreation/ProjectCreation.tsx [hunk 7]
```diff
@@ -12,7 +8,8 @@ import StatusCircle, {
 
 import styles from './ProjectCreation.module.scss';
 import useProject from 'Graphql/hooks/useProject';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
+import { newProject } from '../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/ProjectCreation/ProjectCreation.tsx:12]:
```suggestion
import { newProject } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/Projects/Projects.tsx [hunk 8]
```diff
@@ -5,22 +5,18 @@ import {
...
 import React, { useState } from 'react';
-import { loader } from 'graphql.macro';
 import styles from './Projects.module.scss';
 import useProjectFilters from 'Graphql/client/hooks/useProjectFilters';
-import { useQuery } from '@apollo/client';
+import { useQuery, useReactiveVar } from '@apollo/client';
 
-const GetProjectsQuery = loader('Graphql/queries/getProjects.graphql');
+import GetProjectsQuery from 'Graphql/queries/getProjects';
+import { projectFilters } from '../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/Projects.tsx:19]:
```suggestion
import { projectFilters } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/Users/Users.tsx [hunk 9]
```diff
@@ -19,18 +15,18 @@ import { AccessLevel } from 'Graphql/types/globalTypes';
...
 import styles from './Users.module.scss';
 import useBoolState from 'Hooks/useBoolState';
-import { useQuery } from '@apollo/client';
+import { useQuery, useReactiveVar } from '@apollo/client';
 import useUser from 'Graphql/hooks/useUser';
 import UsersTable from './components/UserList/UsersTable';
 
-const GetUsersQuery = loader('Graphql/queries/getUsers.graphql');
+import GetUsersQuery from 'Graphql/queries/getUsers';
+import { userSettings } from '../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Users/Users.tsx:25]:
```suggestion
import { userSettings } from 'Graphql/client/cache';
```

File: app/ui/src/Pages/Users/components/UserFiltersAndActions/UserActions.tsx [hunk 10]
```diff
@@ -1,17 +1,13 @@
...
 
 import { AccessLevel } from 'Graphql/types/globalTypes';
 import { UserSelection } from 'Graphql/client/models/UserSettings';
 import cx from 'classnames';
-import { get } from 'lodash';
 import styles from './UserFiltersAndActions.module.scss';
-import { useQuery } from '@apollo/client';
+import { useReactiveVar } from '@apollo/client';
 import useUserSettings from 'Graphql/client/hooks/useUserSettings';
+import { userSettings } from '../../../../Graphql/client/cache';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Users/components/UserFiltersAndActions/UserActions.tsx:10]:
```suggestion
import { userSettings } from 'Graphql/client/cache';
```

**Review by marco-gattelli — APPROVED** [review]

File: app/ui/src/Components/ProjectsBar/components/ProjectsFilter/ProjectsFilter.tsx (hunk 1 above)
Comment (mgs95) [reply app/ui/src/Components/ProjectsBar/components/ProjectsFilter/ProjectsFilter.tsx:12]: Done

File: app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.tsx (hunk 2 above)
Comment (mgs95) [reply app/ui/src/Components/ProjectsBar/components/ProjectsOrder/ProjectsOrder.tsx:11]: Done

File: app/ui/src/Pages/NewProject/NewProject.tsx (hunk 3 above)
Comment (mgs95) [reply app/ui/src/Pages/NewProject/NewProject.tsx:24]: Done

File: app/ui/src/Pages/NewProject/pages/RepositoryDetails/RepositoryDetails.tsx (hunk 4 above)
Comment (mgs95) [reply app/ui/src/Pages/NewProject/pages/RepositoryDetails/RepositoryDetails.tsx:6]: Done

File: app/ui/src/Pages/NewProject/pages/RepositoryDetails/components/InternalRepository/InternalRepository.tsx (hunk 5 above)
Comment (mgs95) [reply app/ui/src/Pages/NewProject/pages/RepositoryDetails/components/InternalRepository/InternalRepository.tsx:10]: Done

File: app/ui/src/Pages/NewProject/pages/Summary/Summary.tsx (hunk 6 above)
Comment (mgs95) [reply app/ui/src/Pages/NewProject/pages/Summary/Summary.tsx:9]: Done

File: app/ui/src/Pages/ProjectCreation/ProjectCreation.tsx (hunk 7 above)
Comment (mgs95) [reply app/ui/src/Pages/ProjectCreation/ProjectCreation.tsx:12]: Done

File: app/ui/src/Pages/Projects/Projects.tsx (hunk 8 above)
Comment (mgs95) [reply app/ui/src/Pages/Projects/Projects.tsx:19]: Done

File: app/ui/src/Pages/Users/Users.tsx (hunk 9 above)
Comment (mgs95) [reply app/ui/src/Pages/Users/Users.tsx:25]: Done

File: app/ui/src/Pages/Users/components/UserFiltersAndActions/UserActions.tsx (hunk 10 above)
Comment (mgs95) [reply app/ui/src/Pages/Users/components/UserFiltersAndActions/UserActions.tsx]: Done

**Pull request merged**